components/landing/process-step.vue

```diff
@@ -0,0 +1,38 @@
+<template functional>
+  <div class="container">
+    <img :src="props.img" />
+    <div class="description">
+      <h3 class="title">{{ props.title }}</h3>
+      <p class="subtitle">{{ props.subtitle }}</p>
+      <nuxt-link
```

Comment:
If assigned - should be <a>, not <nuxt-link>. Try clicking on the link - now it redirects relatively

Reply:
But since we are building an SPA, doesn't it make sense to use `<nuxt-link>` which extends `<router-link>` so that links don't open in a new tab?
(I'll fix it for the external link, of course)

Reply:
Only for external, of course